tracer/src/Datadog.Trace/Configuration/ExporterSettings.cs

```diff
@@ -262,7 +268,12 @@ private void ConfigureTraceTransport(string agentUri, string tracesPipeName, int
             {
                 TracesTransport = TracesTransportType.WindowsNamedPipe;
                 TracesPipeName = tracesPipeName;
-                SetAgentUri(agentHost ?? DefaultAgentHost, agentPort ?? DefaultAgentPort); // this one can throw
+
+                // Still build the Uri no matter what the transport as we send it in the http message
```

Comment:
This comment doesn't apply since we refactored the UDS work in #2688

Reply:
Thanks I missed that